vue-frontend/src/components/partials/HeaderV2.vue

```diff
@@ -0,0 +1,231 @@
+<template>
+  <div class="nav-container" :style="{ height: navContainerHeight + 'px' }">
+    <nav
+      class="navbar navbar-expand-md navbar-light main-header"
+      :class="{
+        'navbar-sticky': navbarSticky,
+        'navbar-animation': navbarAnimation,
+      }"
+      ref="mainHeader"
+    >
+      <div class="container col-flow">
+        <router-link to="/" replace>
+          <div class="navbar-brand">
+            <img
+              :src="headerLogoImage"
+              class="header-logo-image mt-1"
+              alt="canopas-logo"
+            />
+          </div>
+        </router-link>
+        <div class="navbar-collapse">
+          <ul class="navbar-nav ms-auto mb-2 mb-lg-0">
+            <li class="nav-item-margin">
+              <a
+                class="nav-link v2-title-3-text"
+                href="/"
+                :style="[
+                  activeHomePath
+                    ? {
+                        textDecoration: `underline !important`,
+                        'text-underline-offset': '0.3rem',
+                      }
+                    : { textDecoration: `none` },
+                ]"
+                ><span :class="[activeHomePath ? '' : 'v2-underline-text']"
```

Comment:
We can add all links in an array and display them here in for loop instead of doing them all separately, also add an active class on click of a link.